app/assets/stylesheets/charts.scss

```diff
@@ -62,6 +62,12 @@
   margin: 0px auto 27px auto;
 }
 
+.statistics-pie-placeholder{
+  margin: 12px auto;
+  width: 240px;
+  height: 268px;
+}
+
 .statistics-pie{
```

Comment:
yes, `.statistics-pie` is used for the charts initialization, which is why I suggested adding the placeholder class to use the same layout (see comment below on the html)